provider/pkg/provider/provider.go

```diff
@@ -111,6 +111,7 @@ type cfnProvider struct {
 	cctl *cloudcontrol.Client
 	ec2  *ec2.Client
```

Comment:
looks like this is just used to implement a `getAZs` function. https://github.com/pulumi/pulumi-aws-native/blob/3585ccb59cfb21f6b8591764521c9652783599e1/provider/pkg/provider/provider.go#L511 I wonder if we should drop that in the future.

Reply:
Yeah, probably an easier way of doing this